app/views/movies/show.html.erb

```diff
@@ -1,29 +1,36 @@
-<%# <% require "pry"; binding.pry %> %>
```

Comment:
Updates all the HTML to use attributes and an @movie object instead of prior methods and hashes